PlayedTracksFetcher.cs

```diff
@@ -0,0 +1,130 @@
+using Microsoft.Azure.Functions.Worker;
+
+namespace Synk.Spotify;
+
+public class PlayedTracksFetcher
+{
+    private readonly TokenStore tokenStore;
+    private readonly UserStore userStore;
+    private readonly MusicStore musicStore;
+    private readonly PlayedTracksStore playedTracksStore;
+    private readonly TokenRefresher tokenRefresher;
+    private readonly SpotifyApi api;
+
+    public PlayedTracksFetcher(
+        TokenStore tokenStore,
+        UserStore userStore,
+        MusicStore musicStore,
+        PlayedTracksStore playedTracksStore,
+        TokenRefresher tokenRefresher,
+        SpotifyApi api)
+    {
+        this.tokenStore = tokenStore;
+        this.userStore = userStore;
+        this.musicStore = musicStore;
+        this.playedTracksStore = playedTracksStore;
+        this.tokenRefresher = tokenRefresher;
+        this.api = api;
+    }
+
+    [Function(nameof(PlayedTracksFetcher))]
+    public async Task Run([TimerTrigger("0 0 * * * *")] ScheduleInfo info)
```

Comment:
Can we move this trigger to a configuration somewhere so we don't need to update this code to update it? `0 0 * * * *` can be a default for that config however.

Reply:
I don't think so. I think attribute arguments need to be compiletime-ready. But will have a look